modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java

```diff
@@ -1295,6 +1316,22 @@ private void _testSystemObjectEntry1toMObjectRelatedModelsProviderImpl()
 		}
 	}
 
+	private void _tetsGetRelatedModels(
```

Comment:
I think you mean "testG..." instead of "tetsG.."

Reply:
You're right. Fixed!